civihr_default_theme/scss/partials/_block.scss

```diff
@@ -75,4 +75,13 @@
   .form-submit {
     min-width: 82px;
   }
-}
\ No newline at end of file
+}
+
+// Use yoti button
+.yoti-connect {
+  text-align: center;
+
+  a {
+    font-size: 14px !important;
```

Comment:
I'm no FE dev, but I know important should be used sparingly, is it possible to get the same result with a more specific CSS selector? If not then this is fine.

Reply:
Well, you right! And its mandatory to use `!important` here as the font size is defined as deafult `inline` in the button markup and we need to customize the default font size. Since the inline style has the higher specificity, we must use `!important` to override it form css.